slave.mk

```diff
@@ -97,7 +99,6 @@ ifeq ($(SONIC_INSTALL_DEBUG_TOOLS),y)
 INSTALL_DEBUG_TOOLS = y
 endif
 
-include $(RULES_PATH)/config
```

Comment:
which one fails? why do we need to move the config to above?

Reply:
I changed/commented out ENABLE_SYSTEM_TELEMETRY= y in rules/config. It didnt take effect.

Reply:
I disabled telemetry by commenting out #ENABLE_SYSTEM_TELEMETRY = y in rules/config.
It didnt take effect.

Reply:
it should not be this case. based on the slave.mk is structured, it should not be moved above. I do not understand.